api/src/test/java/uk/gov/dhsc/htbhf/dwp/service/HouseholdVerifierLegacyHouseholdTest.java

```diff
@@ -0,0 +1,131 @@
+package uk.gov.dhsc.htbhf.dwp.service;
+
+import org.junit.jupiter.api.Test;
+import uk.gov.dhsc.htbhf.dwp.entity.legacy.LegacyAdult;
+import uk.gov.dhsc.htbhf.dwp.entity.legacy.LegacyHousehold;
+import uk.gov.dhsc.htbhf.dwp.model.AddressDTO;
+import uk.gov.dhsc.htbhf.dwp.model.PersonDTO;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static uk.gov.dhsc.htbhf.dwp.entity.LegacyHouseholdFactory.aHouseholdWithNoAdultsOrChildren;
+
+public class HouseholdVerifierLegacyHouseholdTest {
+
+    private final HouseholdVerifier householdVerifier = new HouseholdVerifier();
+    private final AddressDTO address = AddressDTO.builder()
+            .addressLine1("742 Evergreen Terrace")
+            .postcode("AA11AA")
+            .build();
+    private final PersonDTO person = PersonDTO.builder()
+            .forename("Lisa")
+            .surname("Simpson")
+            .address(address)
+            .build();
+
+    @Test
+    void shouldReturnTrueWhenPersonMatchesLegacyHousehold() {
+        LegacyAdult adult = LegacyAdult.builder()
+                .forename("Lisa")
+                .surname("Simpson")
+                .build();
+        LegacyHousehold household = aHouseholdWithNoAdultsOrChildren().build().addAdult(adult);
+
+        Boolean response = householdVerifier.detailsMatch(household, person);
+
+        assertTrue(response);
```

Comment:
I'd use AssertJ for the assertions in this test, they yield much better messages is the tests fail.

Reply:
assertJ also reads better in the test itself

Reply:
Done.